main.go

```diff
@@ -134,6 +134,10 @@ func updateNDK(revision string) error {
 		return err
 	}
 
+	if err := unWrapNDKDirIfNeeded(ndkHome, revision); err != nil {
```

Comment:
`installedNDKVersion` in line 115 needs the correct NDK path to be able to detect the version properly. make sure to have the needed `ndkPath` ready at that point, not after that.

Reply:
As discussed, this could happen due to previous misconfiguration, we should not do this.